src/bin/entry_completion.rs

```diff
@@ -0,0 +1,110 @@
+extern crate glib;
```

Comment:
Please add a small explanation of what's going on in this example at the top of the file. :)